tests/util_test.py

```diff
@@ -180,6 +180,23 @@ def test_placeholder(self):
     self.assertEqual(ph1.name, "geph__a1_0")
     self.assertEqual(ph2.name, "geph")
 
+  def test_identity(self):
+    tf_g = tf.Graph()
+    with tf_g.as_default():
+      c = tf.constant(42)
+      i1 = tf.identity(c, name="identity_tf")
+
+    g = gde.Graph(tf_g)
+    i2 = gde.util.make_identity(g, "identity_gde", g.get_tensor_by_name(c.name))
+
+    with g.to_tf_graph().as_default():
+      with tf.Session() as sess:
+        result1 = sess.run(i1.name)
+        result2 = sess.run(i2.name)
+      # TODO: test fails
+      print(tf.get_default_graph().as_graph_def())
+    self.assertEqual(result1, result2)
```

Comment:
@frreiss this fails because the created identity op return `None` for result2. The graph def output looks the same, and I'm not sure what is wrong, could you take a look at `util.make_identity` when you have a sec? thanks!

Reply:
`i2` is a Node, not a tensor. So line 195 should be:
```
result2 = sess.run(i2.output(0).name)
```